translations/NDRC Welsh Translations.csv

```diff
@@ -47,11 +47,6 @@ find.address	Find address	Dod o hyd i gyfeiriad
 site.service_name	Apply for repayment of import duty and import VAT	Gwneud cais am ad-daliad o dollau mewnforio a TAW mewnforio
 site.textarea.char_limit	(Limit is {0} characters)	(Y terfyn yw {0} o gymeriadau)
 site.govuk	GOV.UK	GOV.UK/CYMRAEG
-footer.cookies	Cookies	Cwcis
```

Comment:
Nice.  Did you run "utils/syncTranslations" to regenerate the welsh messages file?

Reply:
In fact, you didn't need to remove the definitions here.  Just remove from messages.en and run the sync script

Reply:
I cheated - as they where all removals and I could guess what would happen, I manually deleted them.  My bad :(